src/main/java/net/starlark/java/eval/StarlarkInt.java

```diff
@@ -39,6 +39,8 @@
   private static final Int32[] smallints = new Int32[100_000];
 
   static final StarlarkInt ZERO = StarlarkInt.of(0);
+  private static final StarlarkInt ONE = StarlarkInt.of(1);
```

Comment:
Move these two constants to just before where they're needed, in the function body. The operations are fast (array lookup) and rarely used (big * big).

Reply:
Moving constants in function body won't allow identity comparison, but that is probably not a big deal either. I'll move.

Reply:
You're right; let's revert to static final. Sorry.